packages/mantine/src/styles/Pagination.module.css

```diff
@@ -1,3 +1,8 @@
+.root {
+    --pagination-active-bg: var(--mantine-primary-color-light);
+    --pagination-active-color: var(--mantine-primary-color-filled);
```

Comment:
What figma has:
<img width="739" alt="image" src="https://github.com/user-attachments/assets/28e6df24-d56c-4048-a542-ec78445d6e60" />
What the demo has:
<img width="439" alt="image" src="https://github.com/user-attachments/assets/616cae6d-aa95-4c99-95eb-da522f7b3ced" />

Does that mean we should change `--pagination-control-fz` to 14px, or it means we should be setting the prop `size: 'sm'` by default in the theme?

Reply:
I'm reverting this after Gustavo correction